utils/providers.py

```diff
@@ -368,7 +368,7 @@ def clear_provider_by_type(prov_class, validate=True):
         logger.info(' Providers exist, so removing all {} providers'.format(prov_class))
         paginator.results_per_page('100')
         sel.click(paginator.check_all())
-        toolbar.select('Configuration', 'Remove {} Providers from the VMDB'.format(string_name),
+        toolbar.select('Configuration', 'Remove {} Providers'.format(string_name),
```

Comment:
@psav according to this commit, it is general change for all: https://github.com/ManageIQ/manageiq/commit/4cdf4bb3fc6c53b132429f0830370c431fc631f6